src/core/common/CustomLayout.js

```diff
@@ -50,4 +54,11 @@ const CustomLayout = ({ classes, ...rest }) => {
     );
 };
 
-export default withStyles(styles)(CustomLayout);
+const mapStateToProps = state => {
+    const isContrastMode = get(state, 'custom.contrastMode.data', false);
```

Comment:
don't use `get` here, it's useless.

Reply:
I can't remove get() here. 

Contrast mode is non-core feature. And this feature can be absent in project. In this case _state.custom.contrastMode.data_ will return error.

Because current file is located in core path. So, **isContrastMode** should be defined correctly in both cases.

Reply:
you're right, thanks for the explanation.